### ISSUE TITLE
Button to add new Countries to Zones is not working

### ISSUE THREAD

**Issue by thochoertler** [vendure-ecommerce/vendure#3212]
**Describe the bug**
When clicking on the "+ Add Countries to ..." nothing happens and a lot of errors are appearing in the browser console.

Error message in Chrome:
```
query-result.ts:209 ERROR TypeError: Cannot convert undefined or null to object
    at Function.entries (<anonymous>)
    at get sortedColumns (data-table2.component.ts:171:42)
    at get visibleSortedColumns (data-table2.component.ts:165:21)
    at template (data-table2.component.html:22:40)
```

Error message in Safari:
```
[Error] ERROR – TypeError: Object.entries requires that input parameter not be null or undefined — data-table2.component.ts:171
TypeError: Object.entries requires that input parameter not be null or undefined — data-table2.component.ts:171
	handleError (main.a81808e064adad37.js:3246:525988)
	run (polyfills.ca6223cd90c925dc.js:1:20893)
	_tick (main.a81808e064adad37.js:3246:667867)
	tick (main.a81808e064adad37.js:3246:667699)
	(anonymous function) (main.a81808e064adad37.js:3246:670380)
	onInvoke (main.a81808e064adad37.js:3246:562356)
	run (polyfills.ca6223cd90c925dc.js:1:20893)
	next (main.a81808e064adad37.js:3246:670350)
	next (main.a81808e064adad37.js:3246:187761)
	_next (main.a81808e064adad37.js:3246:187431)
	next (main.a81808e064adad37.js:3246:187118)
	(anonymous function) (main.a81808e064adad37.js:3246:185077)
	b (main.a81808e064adad37.js:3246:213750)
	next (main.a81808e064adad37.js:3246:184912)
	emit (main.a81808e064adad37.js:3246:527480)
	Fp (main.a81808e064adad37.js:3246:563350)
	j3 (main.a81808e064adad37.js:3246:563793)
	onInvokeTask (main.a81808e064adad37.js:3246:562297)
	runTask (polyfills.ca6223cd90c925dc.js:1:21543)
	invokeTask (polyfills.ca6223cd90c925dc.js:1:27324)
```

<img width="1512" alt="Screenshot 2024-11-15 at 14 44 29" src="https://github.com/user-attachments/assets/9442e46c-6381-4018-864b-5f7376af137a">

**To Reproduce**
1. Go to 'Settings > Zones'
2. Click on 'View Members' on one item in the table
3. Click on '+ Add Countries to ...'

**Expected behavior**
After clicking the button it should be possible to select and add countries to one zone.

**Environment (please complete the following information):**
 - @vendure/core version: 3.0.6
 - Nodejs version 20.18.0
 - Database (mysql/postgres etc): postgres

**Comment by ashishkpaul** [comment]
Not resolved in v3.07, still having the same issue

**Comment by michaelbromley** [comment]
Thanks for the notice. I'm publishing a new patch which fixes this.
